### DISCUSSION TITLE
water mark so strong

### DISCUSSION THREAD

**Discussion by sghong** [sumatrapdfreader/sumatrapdf#4455] [General]
Is there way to a little of blur alike adobe reader as below ?

![image](https://github.com/user-attachments/assets/b7df4c7d-9714-4c85-a7c5-8beefa5863d0)

this is sumatrapdf.

![image](https://github.com/user-attachments/assets/b5f0520c-4bf4-4235-ab1b-eff90fffcc7f)

**Comment by GitHubRulesOK** [comment]
Hard to be sure from an image however, since the watermark is seemingly showing in foreground compared to Acrobat appears to be background? There is possibly an error in how the object is read or defined. A sample file would need to be provided for analysis.

**Comment by kjk** [comment]
It looks like a bug. Try with latest [pre-release](https://www.sumatrapdfreader.org/prerelease).

If it still happens, you can [open an issue](https://www.sumatrapdfreader.org/docs/How-to-submit-bug-reports). Must include a sample document.
